pyro/util.py

```diff
@@ -162,6 +162,8 @@ def set_rng_seed(rng_seed):
     :param int rng_seed: The seed value.
     """
     torch.manual_seed(rng_seed)
+    if torch.cuda.is_available():
```

Comment:
As of PyTorch 0.3 this was not necessary: `torch.manual_seed()` also calls `torch.cuda.manual_seed()`. Adam recommended we remove this. Has anything changed?

Reply:
Hmm..I probably read some outdated documentation in that case. Let me check and remove it if not needed.